### PR TITLE
DB-10187 add base_name, base_schema to sysibm.systables

### PR DESCRIPTION
**Pull request by yxia92**

### CODE REVIEW

**Comment by yxia92** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

File: splice_machine/src/test/java/com/splicemachine/derby/impl/sql/catalog/ViewsInSysIbmIT.java [hunk 1]
```diff
@@ -189,10 +189,11 @@ public void testColumnDefault() throws Exception {
...
         methodWatcher.executeUpdate(format("create synonym %s.S5 for t5", CLASS_NAME));
-        String sqlText = format("select name, creator, type, colcount, keycolumns, keyunique, codepage from sysibm.systables where CREATOR='%s' and name='S5'", CLASS_NAME);
-        String expected = "NAME |    CREATOR     |TYPE |COLCOUNT |KEYCOLUMNS | KEYUNIQUE |CODEPAGE |\n" +
-                "-------------------------------------------------------------------------\n" +
-                " S5  |VIEWSINSYSIBMIT |  A  |    0    |     0     |     0     |    0    |";
+        String sqlText = format("select * from sysibm.systables where CREATOR='%s' and name in ('S5', 'T5')", CLASS_NAME);
+        String expected = "NAME |    CREATOR     |TYPE |COLCOUNT |KEYCOLUMNS | KEYUNIQUE |CODEPAGE | BASE_NAME |  BASE_SCHEMA   |\n" +
+                "------------------------------------------------------------------------------------------------------\n" +
+                " S5  |VIEWSINSYSIBMIT |  A  |    0    |     0     |     0     |    0    |    T5     |VIEWSINSYSIBMIT |\n" +
+                " T5  |VIEWSINSYSIBMIT |  T  |   13    |     3     |     0     |  1208   |   NULL    |     NULL       |";
```
Comment (ascend1) [inline splice_machine/src/test/java/com/splicemachine/derby/impl/sql/catalog/ViewsInSysIbmIT.java:196]: Is it possible in splice for now that `BASE_NAME` and `BASE_SCHEMA` are not null?

**Review by ascend1 — APPROVED** [review]

File: splice_machine/src/test/java/com/splicemachine/derby/impl/sql/catalog/ViewsInSysIbmIT.java (hunk 1 above)
Comment (yxia92) [reply splice_machine/src/test/java/com/splicemachine/derby/impl/sql/catalog/ViewsInSysIbmIT.java:196]: Base_name and base_schema are only meaningful for alias/synonym that points to a table/view/alias, which refer to the base table/view/alias and the schema the referred object resides in. Splice currently does not have these two columns directly, but the relevant info are in the sys.sysaliases table in the column aliasinfo. For other database objects, these two fields are not applicable, so they should stay as NULL.

**Review by msirek — APPROVED** [review]

**Pull request merged**